declarations/Facebook.json

```diff
@@ -15,12 +15,18 @@
       "executeClientScripts": true
     },
     "Privacy Policy": {
-      "fetch": "https://www.facebook.com/about/privacy",
+      "fetch": "https://mbasic.facebook.com/privacy/policy/printable/",
```

Comment:
Why did you change the fetch url while the document he's still available on https://www.facebook.com/privacy/policy/ ?

Reply:
The printable version was easier to inspect and select parts to crawl, but if that is not best practice to change the url that we crawl, I can change it back

Reply:
Quite the contrary, I think using printable versions is a great way to simplify selection, reduce use of filters, and improve stability! 👏 We could probably try disabling `executeClientScripts` on this version 🙂

Reply:
@MattiSG Currently the privacy policy does not have `executeClientScripts` should I add that and set it to false? 

Or do you mean to remove `executeClientScripts` from ToS since that is a plaintext page

Reply:
> should I add that and set it to false?

You don't need to do that since [the default is `false`](https://github.com/OpenTermsArchive/contrib-declarations/blob/main/CONTRIBUTING.md#executeclientscripts) 🙂

> do you mean to remove `executeClientScripts` from ToS since that is a plaintext page

It is worth checking if your URL change also enables removing `executeClientScripts` from other pages, yes!